h2c.go

```diff
@@ -116,7 +116,11 @@ func setupH2cHijacker(req Request, rw http.ResponseWriter) (http.ResponseWriter,
 
 	h2c := &h2cInfo{
 		conns: mapset.NewSet(),
-		h2s:   &http2.Server{}}
+		h2s: &http2.Server{
+			// We're copying envoy and grpc by setting this to the max uint32.
+			// The Go default is 250 which is not ideal for long lived streaming requests.
+			MaxConcurrentStreams: 1<<32 - 1,
```

Comment:
I know this is equivalent so might seem a bit nitpicky, but I think it would read more easily if you set this to `math.MaxUint32`